ui/admin/tests/e2e/tests/auth-method.spec.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/* eslint-disable no-undef */
+const { test } = require('@playwright/test');
+const { execSync } = require('child_process');
+const { authenticatedState } = require('../helpers/general');
+const {
+  authenticateBoundaryCli,
+  checkBoundaryCli,
+  deleteOrg,
+} = require('../helpers/boundary-cli');
+const {
+  createNewOrg,
+  createNewAuthMethod,
+  addAccountToAuthMethod,
+  setPasswordToAccount,
+  createNewUser,
+  addAccountToUser,
+} = require('../helpers/boundary-ui');
+
+test.use({ storageState: authenticatedState });
+
+test.beforeAll(async () => {
+  await checkBoundaryCli();
+});
+
+test('Verify new auth-method can be created and assigned to users', async ({
+  page,
+}) => {
+  await page.goto('/');
+  let org;
+  try {
+    const orgName = await createNewOrg(page);
+    await authenticateBoundaryCli();
+    const orgs = JSON.parse(execSync('boundary scopes list -format json'));
+    org = orgs.items.filter((obj) => obj.name == orgName)[0];
+
+    await createNewAuthMethod(page, 'UI Test Auth Method');
+    await addAccountToAuthMethod(
+      page,
+      'UI Test Account',
+      'test-user',
+      'password'
+    );
+    await setPasswordToAccount(page, 'password2');
+    await createNewUser(page, 'UI Test User');
+    await addAccountToUser(page);
+  } finally {
```

Comment:
Not sure how you want to handle checks for deletion. Should we do deletes for user, account, auth methods, in this test or create a separate a test that will create and delete everything?

Reply:
I'd rather have a separate test or set of tests for resources deletion scenarios.
In such scenarios we would not need to create the Boundary resources via UI; CLI would be more efficient for the test data preparation stage of the test. Once the test data is prepared, we'd delete the Boundary resources one by one using UI. 

This approach would help achieve two things:

1. The delete test would concentrate on verifying deletion of the resources via UI only
2. The auth-methods test would concentrate on creating auth-methods related resources via UI only and remain relatively small for this test level.